docs/EN_US/ECLStandardLibraryReference/SLR-Mods/DfuPlusExec.xml

```diff
@@ -23,36 +23,39 @@
         <row>
           <entry><emphasis>commandline</emphasis></entry>
 
-          <entry>A null-terminated string containing the DFUPlus.exe command
-          line to execute. The valid arguments are documented in the Client
-          Tools manual, in the section describing the DfuPlus.exe
+          <entry>A null-terminated string containing the DFUPlus command line
+          to execute. The valid arguments are documented in the Client Tools
+          manual, in the section describing the Command Line DFU
           program.</entry>
         </row>
       </tbody>
     </tgroup>
   </informaltable>
 
   <para>The <emphasis role="bold">DfuPlusExec</emphasis> action executes the
-  specified <emphasis>commandline</emphasis> just as the DfuPLus.exe program
-  would do. This simply allows you to have all the functionality of the
-  DfuPLus.exe program available within your ECL code.</para>
+  specified <emphasis>commandline</emphasis> just as the DfuPlus executable
+  program would. This allows you to have all the functionality of DfuPlus
+  available within your ECL code.</para>
+
+  <para>Unless you need to access a foreign instance of the platform, the
+  server= parameter for DfuPlus should be omitted, which then defaults to the
+  value contained in the lib_system.ws_fs_server attribute. </para>
```

Comment:
It actually doesn't look at ws_fs_server:
cloud:
  1. looks for
```
      global:
          defaultEsp: eclservices
```  
2. looks through the available local services for one of type "eclservices" (if it can't find one it will look for the eclwatch service, but not sure we want to advertise that).

baremetal:
looks through the local environment for a service named "WsSMC" that is internal to EclWatch.